cmd/notary/tuf.go

```diff
@@ -266,13 +274,89 @@ func verify(cmd *cobra.Command, args []string) {
 	return
 }
 
-func getTransport() *http.Transport {
-	if viper.GetBool("skipTLSVerify") {
-		return &http.Transport{
-			TLSClientConfig: &tls.Config{
-				InsecureSkipVerify: true,
-			},
-		}
+type passwordStore struct {
+	anonymous bool
+}
+
+func (ps passwordStore) Basic(u *url.URL) (string, string) {
```

Comment:
This is to implement the credential store interface, it will be called by the transport if/when basic auth is needed. The url can be used to select between different credentials, in this case I should actually display the hostname to the user so the user knows where their authentication credentials are being sent.

See
https://github.com/docker/distribution/blob/master/registry/client/auth/session.go#L198